src/pages/Download/index.tsx

```diff
@@ -3,9 +3,12 @@ import { colorsV3 } from '@hedviginsurance/brand/dist'
 import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
 import { TopBar } from 'components/TopBar'
 import { Page } from 'components/utils/Page'
+import { SessionTokenGuard } from 'containers/SessionTokenGuard'
+import { SemanticEvents } from 'quepasa'
```

Comment:
Is this a package for sending events to Segments?

Reply:
Yes! I think its made by us but a long time ago